packages/EasyErrorHandler/src/Builders/ExtendedExceptionErrorResponseBuilder.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Builders;
+
+use EonX\EasyErrorHandler\Interfaces\ErrorDetailsResolverInterface;
+use EonX\EasyErrorHandler\Interfaces\ErrorHandlerAwareInterface;
+use EonX\EasyErrorHandler\Interfaces\Exceptions\TranslatableExceptionInterface;
+use EonX\EasyErrorHandler\Interfaces\TranslatorInterface;
+use EonX\EasyErrorHandler\Traits\ErrorHandlerAwareTrait;
+use Throwable;
+
+final class ExtendedExceptionErrorResponseBuilder extends AbstractSingleKeyErrorResponseBuilder implements
+    ErrorHandlerAwareInterface
+{
+    use ErrorHandlerAwareTrait;
+
+    public const DEFAULT_KEY = 'exception';
+
+    private const EXCEPTION_KEY_CLASS = 'class';
+
+    private const EXCEPTION_KEY_FILE = 'file';
+
+    private const EXCEPTION_KEY_LINE = 'line';
+
+    private const EXCEPTION_KEY_MESSAGE = 'message';
+
+    private const EXCEPTION_KEY_TRACE = 'trace';
+
+    /**
+     * @var string[]
+     */
+    private readonly array $exceptionKeys;
+
+    /**
+     * @param null|string[] $exceptionKeys
+     */
+    public function __construct(
+        private readonly ErrorDetailsResolverInterface $errorDetailsResolver,
+        private readonly TranslatorInterface $translator,
+        ?array $exceptionKeys = null,
+        ?string $key = null,
+        ?int $priority = null
+    ) {
+        $this->exceptionKeys = $exceptionKeys ?? [];
+
+        parent::__construct($key, $priority);
+    }
+
+    /**
+     * @return mixed[]|null
+     */
+    protected function doBuildValue(Throwable $throwable, array $data): ?array
+    {
+        // Skip if not verbose
+        if ($this->errorHandler->isVerbose() === false) {
+            return null;
```

Comment:
We call the parent in the `\EonX\EasyErrorHandler\Builders\AbstractSingleKeyErrorResponseBuilder::buildData` base class method in case of the `null` value.

Reply:
Got it, thanks.